packages/react-core/src/components/Modal/ModalBox.tsx

```diff
@@ -23,14 +25,16 @@ export const ModalBox: React.FunctionComponent<ModalBoxProps> = ({
   variant = 'default',
   title = '',
   'aria-label': ariaLabel = '',
-  id,
+  descriptorId,
+  labelId,
   ...props
 }: ModalBoxProps) => (
   <div
     {...props}
     role="dialog"
-    aria-label={title || ariaLabel}
-    aria-describedby={id}
+    aria-label={title || ariaLabel || null}
```

Comment:
should `ariaLabel` come first so that if someone specifies both, the val for `aria-label` would override `title`?